test/e2e/node/pod_resize.go

```diff
@@ -203,24 +243,34 @@ func verifyPodAllocations(pod *v1.Pod, tcInfo []TestContainerInfo) {
 			}()
 		}
 		tc := makeTestContainer(ci)
-		framework.ExpectEqual(c.ResourcesAllocated, tc.ResourcesAllocated)
+		framework.ExpectEqual(w.ResourcesAllocated, tc.ResourcesAllocated)
 	}
 }
 
 func verifyPodStatusResources(pod *v1.Pod, tcInfo []TestContainerInfo) {
-	csMap := make(map[string]*v1.ContainerStatus)
-	for i, c := range pod.Status.ContainerStatuses {
-		csMap[c.Name] = &pod.Status.ContainerStatuses[i]
-	}
-	for _, ci := range tcInfo {
-		cs, found := csMap[ci.Name]
-		gomega.Expect(found == true)
-		tc := makeTestContainer(ci)
-		framework.ExpectEqual(cs.Resources, tc.Resources)
+	if pod.Spec.VirtualMachine == nil {
```

Comment:
For the memory reduce cases, the vm resource status report could take some time to get back due to the  async pattern in libvirt and the hypervisor,  we should add logic to handle this as well, for example, wait for a few round of 10 seconds ( the status reporting period ) as a timeout waiting for desired status of VMs.

Reply:
this is looking at spec. the code for checking status has a 2 minute timeout i think.